.github/workflows/dotnet.yml

```diff
@@ -84,4 +84,61 @@ jobs:
       - name: Publish to GitHub Packages
         run: gh release upload ${{ github.ref_name }} lib/bin/Release/*.nupkg lib/bin/Release/*.snupkg tool/bin/Release/*.nupkg tool/bin/Release/*.snupkg --repo ${{ github.repository }} --clobber
         env:
-          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
\ No newline at end of file
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  docker-build:
```

Comment:
could you please split the build and the publish jobs? to align with the nuget publish